packages/@okta/vuepress-site/docs/guides/oauth-for-okta/use-client-credentials-grant-flow/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Use the Client Credentials grant flow
```

Comment:
Should we warn people of the implications of granting a scope with client credentials selected? That if someone gets ahold of the client secret and client ID, that they can perform the Okta scopes that have been granted to that service app?

Reply:
added link to more information on client credentials

Reply:
Added link for more information to implementing client credentials grant flow.